Gui/opensim/installer_files/Linux/opensim.conf

```diff
@@ -1,14 +1,14 @@
 # ${HOME} will be replaced by user home directory according to platform
-default_userdir="${HOME}/.${APPNAME}/dev"
-default_mac_userdir="${HOME}/Library/Application Support/${APPNAME}/dev"
+default_userdir="${HOME}/.${APPNAME}/@VERSION@"
+default_mac_userdir="${HOME}/Library/Application Support/${APPNAME}/@VERSION@"
 
 # options used by the launcher by default, can be overridden by explicit
 # command line switches
-default_options="--branding opensim -J-Xms24m -J-Xmx64m"
+default_options="--branding opensim -J-Xms24m -J-Xmx1024M -J-Djava.library.path='${progdir}/../sdk/lib' -J-Dopensim.buildnumber=@VERSION@"
 # for development purposes you may wish to append: -J-Dnetbeans.logger.console=true -J-ea
 
 # default location of JDK/JRE, can be overridden by using --jdkhome <dir> switch
-#jdkhome="/path/to/jdk"
+jdkhome="/usr/lib/jvm/java-8-openjdk-amd64/jre"
```

Comment:
Minor comment: Are you sure this needs to be explicitly defined? Feel free to ignore.

Reply:
This is necessary in the (fairly likely) event that the user has eg openjdk-11 installed and set as the default java.